Doc imorovements (#49)
openclarity/docs.openclarity.io: content/docs/concepts/sbom/output-sbom-format/_index.md

weight: 800
---

> The `openclarity-cli analyze` command can format the resulting SBOM into different formats to integrate with another system. The supported formats are:

Reviewer: ```suggestion
The `openclarity-cli scan` command can format the resulting SBOM into different formats to integrate with another system. The supported formats are:
```

| Syft JSON | syft-json |

{{< warning >}}
OpenClarity processes CycloneDX internally, the other formats are supported through a conversion. The conversion process can be lossy due to incompatibilities between formats, therefore in some cases not all fields/information are present in the resulting output.
{{< /warning >}}

> To configure the `openclarity-cli` to use a format other than the default, the `ANALYZER_OUTPUT_FORMAT` environment variable can be used with the configuration name from above:

Reviewer: ```suggestion
To configure the `openclarity-cli` to use a format other than the default, the `sbom.output_format` config parameter can be used with the configuration name from above:
```

```shell

> ANALYZER_OUTPUT_FORMAT="spdx-json" openclarity-cli analyze nginx:latest -o nginx.sbom

Reviewer: ```suggestion
# Create config based on https://github.com/openclarity/openclarity/blob/main/.families.yaml
cat <<EOF > config.yml
sbom:
  enabled: true
  analyzers_list:
    - "syft"
  inputs:
    - input: "/dir-to-scan"
      input_type: "rootfs"
  output_format: "cyclonedx-json"
EOF

# Run scan
openclarity-cli scan --config config.yml
```

Reviewer: you can also add a link to the config file here
